### PR TITLE
New testing suite

### PR DESCRIPTION
**Pull request by simonarocco**
- created factory models for all tables using factory_boy
- finalized conftest to set up and tear down app, client, db and session objects
- finished the test_db_setup file to make sure the mock db was properly set up

### CODE REVIEW

**Review by abeljoseph — COMMENTED** [review]
Code looks great. Will need to set up a mock db for testing with the CI runner.

**Review by abeljoseph — APPROVED** [review]

File: app/database_etl/etl_main.py [hunk 1]
```diff
@@ -302,13 +302,11 @@ def main():
...
-
+    drop_old_entries()
     return
 
 
 if __name__ == '__main__':
     beginning = time()
     main()
     diff = time() - beginning
-    print(diff)
```
Comment (abeljoseph) [inline app/database_etl/etl_main.py:314]: Would be nice to have something to let us know that the ETL ran successfully

File: app/utils/get_filtered_records.py [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-
-from app.serotracker_sqlalchemy import db_session, AirtableSource, db_model_config
-from sqlalchemy import create_engine
 from itertools import groupby
 from functools import reduce
 
+
+from app.serotracker_sqlalchemy import db_session, AirtableSource, db_model_config
+
 # Create engine to connect to whiteclaw database
```
Comment (atatmaja) [inline app/utils/get_filtered_records.py:7]: can we just delete these

File: tests/utils/helper_funcs.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+
+def delete_records(session, records):
+    for record in records:
+        session.delete(record)
+    session.commit()
+    return
+
+
+def insert_record(session):
```
Comment (atatmaja) [inline tests/utils/helper_funcs.py:11]: looks fine to me but maybe we can make this a bit more concise by using a for loop (similar to what we do in the ETL). fine to do this in another PR tho

**Review by atatmaja — APPROVED** [review]
LGTM, just a few nitpicks

File: tests/utils/helper_funcs.py (hunk 3 above)
Comment (atatmaja) [reply tests/utils/helper_funcs.py:11]: @simonarocco actually I'll just address this. need to make some changes to this anyways bc of estimate pins related schema updates
